src/reducers/index.js

```diff
@@ -8,8 +8,10 @@ import { INCREMENT, DECREMENT } from '../actions';
 export default (count = 0, action) => {
   switch (action.type) {
     case INCREMENT:
+    return count=count+1;
```

Comment:
You don't need to set count `=` to the new value. You can simply write `return count + 1`.
You could also have used `++count` and `--count` here. Do you know why?